fileversion.go

```diff
@@ -218,3 +240,57 @@ func (self *FileVersion) GetFixedFileInfo() (*FixedFileInfo, error) {
 		FileSubtype: VerFileSubtype(ffi.FileSubtype),
 	}, nil
 }
+
+func (self *FileVersion) GetFileInfoTranslations() ([]FileInfoTranslation, error) {
+	var fit *FileInfoTranslation
+	var len uint32
+	err := wrappers.VerQueryValue(
+		&self.data[0],
+		syscall.StringToUTF16Ptr(`\VarFileInfo\Translation`),
+		(**byte)(unsafe.Pointer(&fit)),
+		&len)
+	if err != nil {
+		return nil, NewWindowsError("VerQueryValue", err)
+	}
+
+	result := make([]FileInfoTranslation, 0)
+	if len == 0 {
+		return result, nil
+	}
+
+	ti := fit
+	for i := 0; i < int(len/uint32(unsafe.Sizeof(*ti))); i++ {
```

Comment:
Let's factor out `int(len/uint32(unsafe.Sizeof(*ti)))` into a variable so that we don't have to re-evaluate it every time that we iterate through the loop.